homeassistant/components/light/hue.py

```diff
@@ -166,6 +169,20 @@ def update_lights():
             add_devices(new_lights)
 
     _CONFIGURED_BRIDGES[socket.gethostbyname(host)] = True
+
+    # create a service for calling run_scene directly on the bridge,
+    # used to simplify automation rules.
+    def hue_activate_scene(call):
+        """Service to call directly directly into bridge to set scenes."""
+        args = {key: value for key, value in call.data.items() if key in
+                ("group_name", "scene_name")}
+        bridge.run_scene(args['group_name'], args['scene_name'])
+
+    descriptions = load_yaml_config_file(
+        os.path.join(os.path.dirname(__file__), 'services.yaml'))
+    hass.services.register(DOMAIN, SERVICE_HUE_SCENE, hue_activate_scene,
+                           descriptions.get(SERVICE_HUE_SCENE))
```

Comment:
Can you add a service schema? [Example](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/media_player/sonos.py#L121)